### DISCUSSION TITLE
how to custom doc and blog template?

### DISCUSSION THREAD

**Discussion by lkzwc** [facebook/docusaurus#8361] [Q&A]
for example,show blog like this

![image](https://user-images.githubusercontent.com/84896877/203200387-e2b909ed-bf0d-4982-978b-78c0083726fe.png)

**Comment by slorber** [comment]
You can use swizzle, as documented

Look at personal websites showcase with open-source blogs for inspirations: https://docusaurus.io/showcase?tags=personal

Example: https://joshcena.com/blog/

**Reply by lkzwc** [reply]
it `s how to reference ? can you provide some steps?
